docs/source/en/perf_infer_gpu_one.md

```diff
@@ -234,64 +180,41 @@ inputs = tokenizer(input_text, return_tensors="pt").to("cuda")
 print(tokenizer.decode(outputs[0], skip_special_tokens=True))
 ```
 
-If you see a bug with a traceback saying 
+If you see a bug with the traceback below, try using nightly version of PyTorch which may have broader coverage for FlashAttention:
 
 ```bash
-RuntimeError: No available kernel.  Aborting execution.
-```
+RuntimeError: No available kernel. Aborting execution.
 
-try using the PyTorch nightly version, which may have a broader coverage for Flash Attention:
-
-```bash
+# install PyTorch nightly
 pip3 install -U --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu118
 ```
 
-Or make sure your model is correctly casted in float16 or bfloat16
-
-
-Have a look at [this detailed blogpost](https://pytorch.org/blog/out-of-the-box-acceleration/) to read more about what is possible to do with `BetterTransformer` + SDPA API.
-
-## `bitsandbytes` integration for FP4 mixed-precision inference
-
-You can install `bitsandbytes` and benefit from easy model compression on GPUs. Using FP4 quantization you can expect to reduce up to 8x the model size compared to its native full precision version. Check out below how to get started.
+## bitsandbytes
 
-<Tip>
-
-Note that this feature can also be used in a multi GPU setup.
-
-</Tip>
-
-### Requirements [[requirements-for-fp4-mixedprecision-inference]]
+bitsandbytes is a quantization library that includes support for 4-bit and 8-bit quantization. Quantization reduces your model size compared to its native full precision version, making it easier to fit large models onto GPUs with limited memory.
 
-- Latest `bitsandbytes` library
-`pip install bitsandbytes>=0.39.0`
+Make sure you have the following libraries installed:
 
-- Install latest `accelerate` from source
-`pip install git+https://github.com/huggingface/accelerate.git`
+```bash
+pip install bitsandbytes>=0.39.0
 
-- Install latest `transformers` from source
-`pip install git+https://github.com/huggingface/transformers.git`
+# install Accelerate and Transformers from source
+pip install git+https://github.com/huggingface/accelerate.git
+pip install git+https://github.com/huggingface/transformers.git
```

Comment:
```suggestion
```
I would say no need now to tell users to install accelerate and transformers from source